sw/device/silicon_creator/mask_rom/mask_rom_start.S

```diff
@@ -148,17 +149,23 @@ _mask_rom_start_boot:
   // The following sequence enables the minimum level of entropy required to
   // initialize memory scrambling, as well as the entropy distribution network.
   // FIXME: Enable entropy complex - this is not the full enable.
-  li   a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
-  li   t0, 0x2
-  sw   t0, ENTROPY_SRC_CONF_REG_OFFSET(a0)
+  li a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
+  li t0, (2 << ENTROPY_SRC_CONF_ENABLE_OFFSET) // LFSR mode.
```

Comment:
Can you open an issue to remove LFSR mode in the future? Currently, this helps to facilitate testing, but it should not be part of production code as this bypasses entropy.

Reply:
Done (https://github.com/lowRISC/opentitan/issues/7221).